RPG-game-Hung/Readme.md

```diff
@@ -0,0 +1,15 @@
+This game is a classical fight game! It can be an old fashioned slap fight, or use magic or item
```

Comment:
Utilize [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) syntax to make a better documentation